README.md

```diff
@@ -1,85 +1,49 @@
-## MLTSHP
+# MLTSHP
 
-### Setup Development / Testing environment
+## Overview 
 
-Create settings.py from settings.example.py
+Want to get this set up and make sure it's all working right? Run the following script:
 
-* Update path for `uploaded_files`
-* Add valid AWS bucket reference (`aws_bucket`)
-* Add valid AWS key and secret (`aws_key`, `aws_secret`)
-* Add valid Recaptcha key and secret (`recaptcha_private_key`,
-  `recaptcha_public_key`)
-* Add valid Twitter API keys (`twitter_consumer_key`,
-  `twitter_consumer_secret`, `twitter_access_key`,
-  `twitter_access_secret`)
-
-#### Virtualenv
-
-* Install virtualenv, pip, then run:
-
-    virtualenv env
-    source env/bin/activate
-    pip install -r requirements.txt
-
-You may need to tweak your library path for OpenSSL when
-compiling the mysql module. This worked for me:
-
-    export LDFLAGS="-L/usr/local/opt/openssl/lib"
+    bin/setup
```

Comment:
Can we get a brief summary of what `setup` does (viz. install mysql, install rvm, &c)?